mlflow/pipelines/steps/predict.py

```diff
@@ -71,6 +70,7 @@ def _validate_and_apply_step_config(self):
         self.tracking_config = TrackingConfig.from_dict(self.step_config)
         self.registry_uri = self.step_config.get("registry_uri", None)
         self.skip_data_profiling = self.step_config.get("skip_data_profiling", False)
+        self.force_overwrite = self.step_config.get("force_overwrite", False)
```

Comment:
`force_overwrite` sounds aggressive. Maybe just call it `overwrite_output`?

Related, sorry for the late suggestion, does it make sense to just adopt the Spark SaveMode option here? Since we always use Spark to save the output, instead of a `True/False` option, we allow user to set all available SaveMode, defaulting to "default".
https://spark.apache.org/docs/latest/sql-data-sources-load-save-functions.html#save-modes

In this case we don't have to think about when to overwrite and when not. Just add a link to Spark in the doc solves the problem.

Reply:
Sounds good. I will add a conf for the spark write mode.